recipes/javafx/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# dectect macOS
+IS_MAX_OS=0
+if [ ! -z "${OSX_ARCH}" ]; then
+  IS_MAX_OS=1
+fi
+
+# stop on error
+set -eu -o pipefail
+
+# configure build environment
+export PKG_CONFIG_PATH="${BUILD_PREFIX}/lib/pkgconfig/:${PKG_CONFIG_PATH}"
+if [ ${IS_MAX_OS} -eq 0 ]; then
+  ln -s "${GCC}" "${BUILD_PREFIX}/bin/gcc"
+  ln -s "${GXX}" "${BUILD_PREFIX}/bin/g++"
+else # for macOS
+  ln -s "${CXX}" "${BUILD_PREFIX}/bin/gcc"
+  ln -s "${CXX}" "${BUILD_PREFIX}/bin/g++"
```

Comment:
Please do not do this.  Just pass these values in approriately below.  Or if you really need to do this (which I doubt), you have to remove these links at the end of the build

Reply:
I'll try again but I'm pretty sure that it will fail because it can not find the required compiler.

Reply:
Without the symlinks gcc and g++ is not found and the build fails.

I tried to find another solution but I failed.
I think it might be possible to patch the build.gradle (mediaProperties.compiler, variantProperties.compiler, ...?)file but I won't try that any longer. 

If you have any hints how to pass the values in approriately, I can try that.